liberty-ls/src/main/java/io/openliberty/tools/langserver/model/propertiesfile/PropertiesEntryInstance.java

```diff
@@ -0,0 +1,63 @@
+/*******************************************************************************
+* Copyright (c) 2022 IBM Corporation and others.
+*
+* This program and the accompanying materials are made available under the
+* terms of the Eclipse Public License v. 2.0 which is available at
+* http://www.eclipse.org/legal/epl-2.0.
+*
+* SPDX-License-Identifier: EPL-2.0
+*******************************************************************************/
+package io.openliberty.tools.langserver.model.propertiesfile;
+
+import java.util.concurrent.CompletableFuture;
+
+import org.eclipse.lsp4j.Hover;
+import org.eclipse.lsp4j.MarkupContent;
+import org.eclipse.lsp4j.Position;
+
+import io.openliberty.tools.langserver.ls.LibertyTextDocument;
+
+public class PropertiesEntryInstance {
+    private PropertiesKeyInstance propertyKeyInstance;
+    private PropertiesValueInstance propertyValueInstance;
+    private String line;
+    private LibertyTextDocument textDocumentItem;
+    private boolean isComment;
+
+    public PropertiesEntryInstance(String entryLine, LibertyTextDocument textDocumentItem) {
+        this.line = entryLine;
+        this.textDocumentItem = textDocumentItem;
+        if (line.trim().startsWith("#")){
+            isComment = true;
+            return;
+        }
+        isComment = false;
+        int equalsIndex = line.indexOf("=");
+        String propertyKeyInstanceString;
+        String propertyValueInstanceString;
+        if (equalsIndex != -1) {
+            propertyKeyInstanceString = line.substring(0, equalsIndex);
+            propertyValueInstanceString = line.substring(equalsIndex+1);
```

Comment:
Interesting catch, it would seem so. 
I looked at the javadocs for the `substring(beginIndex)` method, and an example seems to show that calling it on that index returns an empty string.
To test, I tried `+2` and that ended up out of bounds, but it's fine with the `+1`.